### PR TITLE
Fixed scene.pick object empty after exiting VR

### PR DESCRIPTION
**Pull request by muyao1987**
Problem recurrence steps:

1.  Click the point entity object. Console. log prints the object value
2.  Click VRButton to enter VR mode
3.  Exit VR mode
4. Viewer. Scene. Pick cannot pick objects, and the printed value is undefined

```
viewer.entities.add({
      position: Cesium.Cartesian3.fromDegrees(116.39224, 30.902853),
      point: {
        color: Cesium.Color.YELLOW,
        pixelSize: 30,
      },
    });

    let handler = new Cesium.ScreenSpaceEventHandler(viewer.scene.canvas);
    handler.setInputAction((event) => {
      let pickedObject =viewer. scene.pick(event.position);
      console.log("pickedObject",pickedObject)  // **Open VR and exit the post-VR mode, and print undefined**

    }, Cesium.ScreenSpaceEventType.LEFT_CLICK);

```

This commit fixes the problem

### CODE REVIEW

**Comment by cesium-concierge** [comment]
Thank you so much for the pull request @muyao1987! I noticed this is your first pull request and I wanted to say welcome to the Cesium community!

The [Pull Request Guidelines](https://github.com/CesiumGS/cesium/blob/main/CONTRIBUTING.md#pull-request-guidelines) is a handy reference for making sure your PR gets accepted quickly, so make sure to skim that.

* :x: Missing `CONTRIBUTORS.md` entry.
    * Please add yourself to the [contributors](https://github.com/marsgis/cesium/blob/main/CONTRIBUTORS.md) file!
* :x: Missing CLA.
    * Please send in a [Contributor License Agreement](https://github.com/CesiumGS/cesium/blob/main/CONTRIBUTING.md#contributor-license-agreement-cla) (CLA) and comment back here to let us know to check this!
    * Maintainers, [here's how to check](https://github.com/CesiumGS/cesium/blob/main/Documentation/Contributors/CLAs/README.md) for a submitted CLA.
* :grey_question: [CHANGES.md](https://github.com/CesiumGS/cesium/blob/main/CHANGES.md) was not updated.
    * If this change updates the public API in any way, please add a bullet point to `CHANGES.md`.
* :grey_question: Unit tests were not updated.
    * Make sure you've [updated tests](https://github.com/CesiumGS/cesium/tree/main/Documentation/Contributors/TestingGuide) to reflect your changes, added tests for any new code, and ran the code coverage tool.

Reviewers, don't forget to make sure that:

- [ ] Cesium Viewer works.
- [ ] Works in 2D/CV.

**Comment by muyao1987** [comment]
It has been handled as required

[x]   Missing `CONTRIBUTORS.md` entry. 
[x]   Missing CLA.
[ x]  Cesium Viewer works.
[x ]  Works in 2D/CV.

**Comment by sanjeetsuhag** [comment]
Thanks for the PR @muyao1987! I don't see your username listed in the CLA list. Did you sign the Individual CLA using this GitHub account?

**Comment by muyao1987** [comment]
> Thanks for the PR @muyao1987! I don't see your username listed in the CLA list. Did you sign the Individual CLA using this GitHub account?

已经提交

**Comment by muyao1987** [comment]
@sanjeetsuhag Hello, have you received the CLA I submitted? And can the problem itself be fixed in this way?

File: Source/Scene/Scene.js [hunk 1]
```diff
@@ -2880,6 +2880,7 @@ Scene.prototype.updateAndExecuteCommands = function (
     mode !== SceneMode.SCENE2D ||
     this._mapMode2D === MapMode2D.ROTATE
   ) {
+    exitWebVRUpdateViewport(this, passState);
```
Comment (sanjeetsuhag) [inline Source/Scene/Scene.js:2883]: I'm not that familiar with VR but having this code run per-tick does not seem correct to me.

Comment (muyao1987) [reply Source/Scene/Scene.js:2883]: The current bug problem is that when entering VR state and exiting again, `viewer.scene.pick(event.position); ` can't pick up the relevant entity  object, and the return value is undefined.


The code I modified is to restore the operation done when entering VR, corresponding to the exitWebrupdateViewPort method.

**Comment by cesium-concierge** [comment]
Thanks again for your contribution @muyao1987! 

No one has commented on this pull request in 90 days. Maintainers, can you review, merge or close to keep things tidy?

 _I'm going to re-bump this in 90 days. If you'd like me to stop, just comment with `@cesium-concierge stop`. If you want me to start again, just delete the comment._

**Comment by cesium-concierge** [comment]
Thanks again for your contribution @muyao1987! 

No one has commented on this pull request in 90 days. Maintainers, can you review, merge or close to keep things tidy?

 _I'm going to re-bump this in 90 days. If you'd like me to stop, just comment with `@cesium-concierge stop`. If you want me to start again, just delete the comment._

**Comment by cesium-concierge** [comment]
Thanks again for your contribution @muyao1987! 

No one has commented on this pull request in 90 days. Maintainers, can you review, merge or close to keep things tidy?

 _I'm going to re-bump this in 90 days. If you'd like me to stop, just comment with `@cesium-concierge stop`. If you want me to start again, just delete the comment._

**Pull request closed without merging**